domain/src/test/java/org/oppia/android/domain/auth/FirebaseAuthWrapperImplTest.kt

```diff
@@ -0,0 +1,3 @@
+package org.oppia.android.domain.auth
+
+class FirebaseAuthWrapperImplTest
```

Comment:
What's this file for?

Reply:
Tests for the prod impl of FirebaseAuthWrapper